docker.md

```diff
@@ -121,7 +121,14 @@ Manages `image`s.
 docker rmi b750fe78269d
 ```
 
-Deletes `image`s.
+Delete `image`.
+
+
+```yml
+docker rmi $(docker images -q) -f
+```
+
+Deletes all images.
```

Comment:
Let's update this to be in one block and change the code style to be `sh` instead of `yml`:

```sh
$ docker rmi b750fe78269d             # delete image
$ docker rmi $(docker images -q) -f   # delete all images
```

and update the text below to say 

    Delete `image`(s).

Reply:
Sure happy with that. LGTM.